src/components/checkbox/__tests__/index.spec.js

```diff
@@ -0,0 +1,94 @@
+import React, {useState, useCallback} from 'react';
+import {render, fireEvent} from '@testing-library/react-native';
+import Checkbox from '../index';
+
+export const testID = 'checkbox';
+const onValueChange = jest.fn();
+
+const TestCase = props => {
+  const {value, onValueChange, ...others} = props;
+
+  const [_value, _setValue] = useState(value);
+  const _onValueChange = useCallback(newValue => {
+    _setValue(newValue);
+    onValueChange?.(newValue);
+  },
+  [_setValue, onValueChange]);
+
+  return <Checkbox {...others} onValueChange={_onValueChange} value={_value} testID={testID}/>;
+};
+
+describe('Checkbox renderer test', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('Default value is false', async () => {
+    const props = {onValueChange};
+
+    const {getByTestId} = render(<TestCase {...props}/>);
+
+    const checkbox = getByTestId(testID);
+    fireEvent.press(checkbox);
+    expect(onValueChange).toHaveBeenCalledTimes(1);
+    expect(onValueChange).toHaveBeenCalledWith(true);
+  });
+
+  it('Send value (false)', async () => {
```

Comment:
Why you need this test? You checked the default (false) in the previous test, are you checking react's passed prop mechanism or there's a different behavior when passing false over a default false?

Reply:
I think I prefer to test explicitly as well as default behaviour, not sure what the best practice, do you want to ask the team?

Reply:
Sure, we can ask them. Anyways Let's leave it in the meantime